packages/date-parser/src/refiners/ambiguity.test.js

```diff
@@ -0,0 +1,16 @@
+import ambiguity from './ambiguity';
+
+describe('ambiguity refiner', () => {
+  it('can handle empty knowValues case', () => {
+    const inputResults = [
+      {
+        start: {
+          knownValues: {},
+        },
+      },
+    ];
+
+    const res = ambiguity.refine(null, inputResults);
+    expect(res).toEqual([]);
+  });
+});
```

Comment:
weird, there was previously no test covering this?  
If it's because Chrono has upgraded and won't return empty knownValues anymore, then we should just remove this refiner.